.pre-commit-config.yaml

```diff
@@ -1,9 +1,7 @@
-exclude: &exclude archive
```

Comment:
The syntax is valid as per the [YAML spec](https://yaml.org/spec/1.2/#id2760658).